### DISCUSSION TITLE
Lazy load images (next/image) within predefined pixel distance.

### DISCUSSION THREAD

**Discussion by LeeBuckle** [vercel/next.js#32228] [Ideas]
### Describe the feature you'd like to request

Would be nice to be able to provide a pixel distance to the Image component that tells it to load lazy images within that many pixels of the viewport.

This would provide more control and allow us to get a bit of head-start on loading some images which can increase UX and mitigate some of the pop-in / white flash.

### Describe the solution you'd like

```
<Image
   src={/images/my-image.jpg}
   width={100}
   height={65}
   distance={50} // load the lazy loaded image within 50 pixels of the viewport.
/>
```

### Describe alternatives you've considered

When manually pre-loading images in the past, this is quite simple and it would be nice to have an option like this by default with Next.

**Comment by miguderp** [comment]
[There's a PR](https://github.com/vercel/next.js/pull/21129) that has been a bit forgotten for that but in the meantime you can check [this solution](https://github.com/vercel/next.js/discussions/24552#discussioncomment-705177) suggested by @dulikkk
